spring-batch-docs/src/main/asciidoc/job.adoc

```diff
@@ -1775,55 +1827,52 @@ public Job footballJob() {
-      immediate shutdown, especially if the execution is currently in
-      developer code that the framework has no control over, such as a
-      business service. However, as soon as control is returned back to the
-      framework, it will set the status of the current
-      `StepExecution` to
-      `BatchStatus.STOPPED`, save it, then do the same
-      for the `JobExecution` before finishing.
-
-
+immediate shutdown, especially if the execution is currently in
+developer code that the framework has no control over, such as a
+business service. However, as soon as control is returned back to the
+framework, it sets the status of the current
+`StepExecution` to
+`BatchStatus.STOPPED`, saves it, and then does the same
+for the `JobExecution` before finishing.
 
 ==== Aborting a Job
 
-A job execution which is `FAILED` can be
-      restarted (if the `Job` is restartable). A job execution whose status is
-      `ABANDONED` will not be restarted by the framework.
-      The `ABANDONED` status is also used in step
-      executions to mark them as skippable in a restarted job execution: if a
-      job is executing and encounters a step that has been marked
-      `ABANDONED` in the previous failed job execution, it
-      will move on to the next step (as determined by the job flow definition
-      and the step execution exit status).
+A job execution that is `FAILED` can be
+restarted (if the `Job` is restartable). A job execution whose status is
+`ABANDONED` cannot be restarted by the framework.
+The `ABANDONED` status is also used in step
+executions to mark them as skippable in a restarted job execution. If a
+job is running and encounters a step that has been marked
+`ABANDONED` in the previous failed job execution, it
+moves on to the next step (as determined by the job flow definition
+and the step execution exit status).
 
 If the process died (`"kill -9"` or server
-      failure) the job is, of course, not running, but the `JobRepository` has
-      no way of knowing because no-one told it before the process died. You
-      have to tell it manually that you know that the execution either failed
-      or should be considered aborted (change its status to
-      `FAILED` or `ABANDONED`) - it's
-      a business decision and there is no way to automate it. Only change the
-      status to `FAILED` if it is not restartable, or if
-      you know the restart data is valid. There is a utility in Spring Batch
-      Admin `JobService` to abort a job execution.
+failure) the job is, of course, not running, but the `JobRepository` has
+no way of knowing because no one told it before the process died. You
+have to tell it manually that you know that the execution either failed
+or should be considered aborted (change its status to
+`FAILED` or `ABANDONED`). This is
+a business decision, and there is no way to automate it. Change the
+status to `FAILED` only if it is not restartable or if
+you know the restart data is valid. Spring Batch
```

Comment:
Not directly related to this pull request, but I guess the mention of `Spring Batch Admin` should be removed, as it was discontinued?

Reply:
I agree. Done in 753a538a6979ab57c321c5bb6f5a5958b9130f5d.